Mathlib/Algebra/Order/AddGroupWithTop.lean

```diff
@@ -194,4 +194,32 @@ instance (priority := 100) toSubtractionMonoid : SubtractionMonoid α where
     intro v
     simp [v] at h
 
+lemma injective_add_ne_top (b : α) (h : b ≠ ⊤) : Function.Injective (fun x ↦ x + b) := by
+  intro x y h2
+  replace h2 : x + (b + -b) = y + (b + -b) := by simp [← add_assoc, h2]
+  simpa only [LinearOrderedAddCommGroupWithTop.add_neg_cancel _ h, add_zero] using h2
+
+lemma strictMono_add_ne_top (b : α) (h : b ≠ ⊤) : StrictMono (fun x ↦ x + b) := by
```

Comment:
I think this names should be `of_ne_top`. And maybe they should also have a `left/right` in them (not sure which).
Maybe @YaelDillies can help with that.

Reply:
`strictMono_add_left_of_ne_top` 😉